calrissian/job.py

```diff
@@ -28,11 +29,29 @@ def k8s_safe_name(name):
     return name.lower().replace('_', '-')
 
 
-def populate_demo_volume_builder_entries(volume_builder):
-    volume_builder.add_persistent_volume_entry('/calrissian/input-data', 'calrissian-input-data')
-    volume_builder.add_persistent_volume_entry('/calrissian/output-data', 'calrissian-output-data')
-    volume_builder.add_persistent_volume_entry('/calrissian/tmptmp', 'calrissian-tmp')
-    volume_builder.add_persistent_volume_entry('/calrissian/tmpout', 'calrissian-tmpout')
+class KubernetesPod(object):
```

Comment:
I'd suggest a name like `KubernetesPodVolumeInspector` or `KubernetesPodInspector` to clarify the purpose of this class - it's not a general purpose Pod class, it exists for the purpose of inspecting the volumes attached to a Pod.